modules/migration/src/main/java/com/sap/xsk/migration/neo/sdk/command/databases/ListDatabasesSdkCommand.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2019-2021 SAP SE or an SAP affiliate company and XSK contributors
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-FileCopyrightText: 2019-2021 SAP SE or an SAP affiliate company and XSK contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.sap.xsk.migration.neo.sdk.command.databases;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import com.sap.xsk.migration.neo.sdk.command.AbstractSdkCommand;
+import com.sap.xsk.migration.neo.sdk.command.SdkCommandGenericArgs;
+import com.sap.xsk.migration.neo.sdk.command.SdkCommandParsedOutput;
+import com.sap.xsk.migration.tooling.MigrationToolExecutor;
+import javax.inject.Inject;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+class ListDatabasesSdkCommand extends AbstractSdkCommand<SdkCommandGenericArgs, ListDatabasesSdkCommandRes> {
+
+  private static final String LIST_DATABASES_COMMAND_NAME = "list-dbs";
+
+  @Inject
+  public ListDatabasesSdkCommand(MigrationToolExecutor migrationToolExecutor) {
+    super(migrationToolExecutor);
+  }
```

Comment:
To be refactored without DI

Reply:
Personally, I prefer to still have DI. I find it easier to maintain the code this way instead of creating new objects and passing them around. I agree that there is no need for separate Guice modules at this time. Take a look at my last commit and we can chat about it if you want